packages/adblocker/tools/generate_compression_codebooks.ts

```diff
@@ -11,6 +10,7 @@ import {
   fullLists,
   hasUnicode,
 } from '../src/index.js';
+import { Smaz } from '../src/compression.js';
```

Comment:
Did you re-run the generation in order to trigger the validation code with the new `Smaz`?